soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -12,6 +12,9 @@ config CORTEX_M_SYSTICK
 	default n if STM32_LPTIM_TIMER
 	default y if !STM32_LPTIM_TIMER
 
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1024 if STM32_LPTIM_TIMER
```

Comment:
Does it depends also on LPTIM clock source  (LSI or LSE)?
In that case, following would be better:
```
	default 1024 if STM32_LPTIM_TIMER AND STM32_LPTIM_CLOCK_LSE

Reply:
or even better:
```
config SYS_CLOCK_TICKS_PER_SEC
	default 1000 if STM32_LPTIM_CLOCK_LSI
	default 1024 if STM32_LPTIM_CLOCK_LSE
	depends on STM32_LPTIM_TIMER 
```

Reply:
Indeed!